game/src/backend/gamestate.rs

```diff
@@ -240,11 +240,15 @@ mod test {
 
     #[test]
     fn test_load_autosave() {
-        let gamestate = GameState::load(false).unwrap();
+        let gamestate = GameState::default();
+        gamestate.save(false).unwrap();
+        let gamestate_loaded = GameState::load(false).unwrap();
```

Comment:
Ich würd hier das ganze nicht zu ner variable machen, solange wir nichts asserten aber is fine